### PR TITLE
Add audio input to qemu in order to match vz

### PR DESCRIPTION
**Pull request by afbjorklund**
PR #1532 

Follow-up:
* #1527

### CODE REVIEW

File: pkg/qemu/qemu.go [hunk 1]
```diff
@@ -600,7 +600,7 @@ func Cmdline(cfg Config) (string, []string, error) {
 		// audio controller
 		args = append(args, "-device", "ich9-intel-hda")
 		// audio codec
-		args = append(args, "-device", fmt.Sprintf("hda-output,audiodev=%s", id))
+		args = append(args, "-device", fmt.Sprintf("hda-duplex,audiodev=%s", id))
```
Comment (AkihiroSuda) [inline pkg/qemu/qemu.go:603]: Could you update the comment in `default.yaml`?

Also wondering if we can have a YAML property to disable the audio input

Comment (AkihiroSuda) [reply pkg/qemu/qemu.go:603]: @afbjorklund WDYT?

Comment (afbjorklund) [reply pkg/qemu/qemu.go:603]: A boolean could be added, not sure how to implement it for vz though

Comment (AkihiroSuda) [reply pkg/qemu/qemu.go:603]: VZ implementation can be another PR

Comment (AkihiroSuda) [reply pkg/qemu/qemu.go:603]: What should we do with this for v0.16.0?

Comment (balajiv113) [reply pkg/qemu/qemu.go:603]: My take would be we can support both for now as by default (as we have support for this in both drivers).

Later if we get any usecase for disabling input we will be able to easily provide at that point of time

Comment (AkihiroSuda) [reply pkg/qemu/qemu.go:603]: For security I'd prefer to disable input by default

Comment (balajiv113) [reply pkg/qemu/qemu.go:603]: Ah yes, i missed the security point. It makes sense.

I will create a PR for vz to support the same.

Comment (afbjorklund) [reply pkg/qemu/qemu.go:603]: ok, so then we are good for qemu ?

**Pull request closed without merging**